hs_core/templates/email/group_membership.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+{% block main %}
+    {% if group_owner %}
+        <p>{% trans "Dear " %}{{ group_owner.first_name }}</p>
+        <p>{% trans "User " %}{{ user.first_name }} has requested to join your HydroShare group ({{ group.name }}). Please click on the link below to accept this request.</p>
```

Comment:
@hyi I guess we can do that. I will wait until some test has been done by @dtarb or @horsburgh to finalize the content of this email notification..